phoenix-core/src/main/java/org/apache/phoenix/execute/SortMergeJoinPlan.java

```diff
@@ -45,10 +45,7 @@
 import org.apache.phoenix.exception.SQLExceptionInfo;
 import org.apache.phoenix.execute.TupleProjector.ProjectedValueTuple;
 import org.apache.phoenix.expression.Expression;
-import org.apache.phoenix.iterate.DefaultParallelScanGrouper;
-import org.apache.phoenix.iterate.MappedByteBufferQueue;
-import org.apache.phoenix.iterate.ParallelScanGrouper;
-import org.apache.phoenix.iterate.ResultIterator;
+import org.apache.phoenix.iterate.*;
```

Comment:
Please do always check your patch before you submit. This is not the right coding style, and such changes should never appear in a patch/pull request.